build.sbt

```diff
@@ -70,7 +70,6 @@ lazy val plugin = project("sbt-paradox-material-theme", file("plugin"))
     addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.4.4"),
     libraryDependencies += "org.jsoup" % "jsoup"      % "1.10.3",
     libraryDependencies += "io.circe" %% "circe-core" % "0.9.3",
-    update := update.dependsOn(theme / publishLocal).value,
```

Comment:
So this was put in deliberately because otherwise you cannot compile/update the project locally.

So I don't what to do here, ir seems like a catch 22

Reply:
@mkurz Does this help trigger your memories wrt https://github.com/sbt/sbt-paradox-material-theme/pull/48#issuecomment-1936133791 ?

Reply:
Sorry, can't remember. Great you solved this meanwhile :+1: